.github/workflows/main.yml

```diff
@@ -0,0 +1,30 @@
+name: build
+on:
+  # run on push but only for the main branch
+  push:
+    branches:
+      - main
```

Comment:
Will need github access to move default branch from master to main.

Reply:
@westeezy added you to the repo.